### DISCUSSION TITLE
Enable slider turnback points to have click tolerance

### DISCUSSION THREAD

**Discussion by nana123126** [ppy/osu#23352] [Ideas]
The turn back point of the slider must be accurately hit, but if the distance between the slider head and the turn back point is very small (if the distance is only 20ms, the OD difficulty is 0), then players can only delay clicking on the slider head for up to 20ms to avoid break combo. However, players can also avoid break combo by 200ms in advance. Obviously, players must deliberately go faster, but this will damage the feel, And at certain times, this operation cannot be done (when the slider is also very close to the previous object). It should be possible to share the 200ms tolerance for the slider's turn back points, even if the player delays by 200ms when clicking on the slider head, it will automatically capture all slider turn back points and slider points on the way.

other:

-If the slider head is too close to the slider turn back point, the turn back point symbol is not visible (official skin effect), which requires players to hold down for a longer time to avoid missing the slider turn back point when seeing a very short slider.

-When the speed of the slider changes, it expresses the change in speed in some form, but it allows the genealogist to choose whether to present it.

-Need to aim at the slider, so aim PP should be added

-Clicking on the slider head should also increase Acc PP.

(machine translation)

**Comment by peppy** [comment]
> -If the slider head is too close to the slider turn back point, the turn back point symbol is not visible (official skin effect), which requires players to hold down for a longer time to avoid missing the slider turn back point when seeing a very short slider.

this was supposed to be addressed in skin changes for the default skin (@arflyte can you advise?)
